haskell/compile.bzl

```
@@ -437,6 +437,12 @@ def _common_compile_module_args(
         format="--bin-path={}/bin",
     ))
 
+    external_tool_paths = [tool[RunInfo] for tool in ctx.attrs.external_tools]
+    args_for_file.add(cmd_args(
+        external_tool_paths,
+        format="--bin-exe={}",
+    ))
```

As discussed on Slack I think this should also work with something like this
```suggestion
    args_for_file.add(cmd_args(
        external_tool_paths,
        format="--bin-path={}",
        parent=1,
    ))
```
without the need for the new `--bin-exe` flag.

---

Thank you for the tip!!